test/build/entry/scss/scss.test.js

```diff
@@ -1,10 +1,8 @@
 /* eslint-disable   node/no-unpublished-require */
-const { run, runInstall } = require('../../../utils/test-utils');
+const { run } = require('../../../utils/test-utils');
 
 describe('entry point', () => {
     it('should support SCSS files', async () => {
-        await runInstall(__dirname);
```

Comment:
Why we remove this?

Reply:
we do it in `prepareSuite.js` this is redundant